Localization/fr-fr.cfg

```
@@ -0,0 +1,199 @@
+		#autoLOC_CustomAsteroids_SciIcyDuna = Vous n'obtenez qu'un échantillon des morceaux organiques lorsque la glace se vaporise.
+		#autoLOC_CustomAsteroids_SciIcyDres = Les gaz s'échappent de l'échantillon. Tu devrais peut-être le mettre dans la glacière.
+
+		#autoLOC_CustomAsteroids_CompCmt = Comète
+		#autoLOC_CustomAsteroids_TitleCmt = Une boule de neige sale
+		#autoLOC_CustomAsteroids_InfoCmt = Ça ressemble à un rocher de l'extérieur, mais avec beaucoup d'eau à l'intérieur.
+		#autoLOC_CustomAsteroids_TagsCmt = comète boule de neige sale
+		#autoLOC_CustomAsteroids_SciCmtSample = Échantillon de comète
+		#autoLOC_CustomAsteroids_SciCmt1 = C'est juste du rock. Peut-être que si vous creusiez plus profondément?
+		#autoLOC_CustomAsteroids_SciCmt2 = Vous parvenez en fait à trouver une poche de glace vierge. Eh bien, vierge! D'accord, c'est plutôt sale en fait.
+		#autoLOC_CustomAsteroids_SciCmtFlying = Vous récupérez un échantillon dans les airs. La comète semble sur le point d'exploser !
+		#autoLOC_CustomAsteroids_SciCmtSun1 = La comète tout entière se sublime. Vous pourriez obtenir un échantillon de *gaz*...
+		#autoLOC_CustomAsteroids_SciCmtSun2 = C'est un miracle qu'il reste tout sauf du rock.
+		#autoLOC_CustomAsteroids_SciCmtMoho = C'est un miracle qu'il reste tout sauf du rock.
+		#autoLOC_CustomAsteroids_SciCmtEve = Vous parvenez à échantillonner un morceau qui n'envoie pas d'eau dans l'espace.
+		#autoLOC_CustomAsteroids_SciCmtKerbin = Vous parvenez à échantillonner un morceau qui n'envoie pas d'eau dans l'espace.
+		#autoLOC_CustomAsteroids_SciCmtDuna1 = La comète a des jets effrayants, mais vous parvenez à retirer un échantillon de glace de son côté nocturne.
+		#autoLOC_CustomAsteroids_SciCmtDuna2 = Les morceaux d'apparence rocheuse ne semblent pas affectés par tout ce qui projette des jets hors de la surface de la comète.
+
+		#autoLOC_CustomAsteroids_MbcName = Astéroïde actif
+		#autoLOC_CustomAsteroids_MbcBlurb = Un certain nombre d'astronomes ont rapporté que cet astéroïde apparaît flou près du périhélie. Ils ont probablement juste besoin de nettoyer à nouveau les lentilles de leur télescope.
+		#autoLOC_CustomAsteroids_SciMbcSample = Échantillon de comète de la ceinture principale
+		#autoLOC_CustomAsteroids_SciMbc1 = Cet astéroïde a quelques dépôts de glace à la surface, probablement intacts depuis la formation de la ceinture d'astéroïdes. Les scientifiques seront ravis d'étudier sa composition isotopique, mais ce serait presque dommage de la laisser fondre.
+		#autoLOC_CustomAsteroids_SciMbc2 = Les meilleurs esprits Kerbal se sont demandé si cet objet était un astéroïde ou une comète. Après avoir pris un échantillon, vous n'êtes toujours pas sûr de 
+
+
+		#autoLOC_CustomAsteroids_GuiClass = Type
+		#autoLOC_CustomAsteroids_GuiDebugResource = Survey Asteroid

```

```suggestion
		#autoLOC_CustomAsteroids_GuiDebugResource = Sondez l'astéroïde
```
(I'm going by a dictionary, so I'm not sure if this is the correct translation of "survey" -- the command in question produces a list of all the resources in the asteroid, along with their amounts, for debugging purposes.)